inst/tests/tests.Rraw

```diff
@@ -12483,7 +12489,266 @@ gs = groupingsets(d, j = sum(val), by = c("a", "b", "c"),
                            character()), id=TRUE)
 test(1961, cb, gs)
 
-# tests 1962 & 1963 reserved for other PRs
+# coverage tests
+## duplicated.R
+opt = options("datatable.old.unique.by.key" = TRUE)
+DT = data.table(x = c(1, 1, 3, 2), key = 'x')
+test(1962.01, duplicated(DT), c(FALSE, TRUE, FALSE, FALSE),
+     warning = 'deprecated option')
+test(1962.02, anyDuplicated(DT), 2L,
+     warning = 'deprecated option')
+options("datatable.old.unique.by.key" = opt$datatable.old.unique.by.key)
+
+test(1962.03, duplicated(DT, fromLast = NA),
+     error = 'must be TRUE or FALSE')
+test(1962.04, duplicated(DT, by = -1L),
+     error = 'Integer values between 1 and ncol')
+test(1962.05, duplicated(DT, by = 'y'),
+     error = 'by specifies column names that do not exist')
+test(1962.06, duplicated(data.table(NULL)), logical(0L))
+
+test(1962.07, unique(DT, incomparables = TRUE),
+     error = 'not used (yet)')
+test(1962.08, unique(DT, fromLast = TRUE),
+     data.table(x = c(1, 2, 3), key = 'x'))
+
+test(1962.09, anyDuplicated(DT, by = NULL), 2L)
+
+## uniqlist.R
+test(1962.10, uniqlist(1:5),
+     error = 'not type list')
+test(1962.11, uniqlist(list()), list(0L))
```

Comment:
@MichaelChirico This is an example where blind aiming for higher code cov can cause an issue. AFAIU `uniqlist` was used more widely before, as of now it is being used in very few places. None of those, AFAICT, requires this case to be handled in the way as it is defined by 1962.11 test. So now in #4372 I am changing this behaviour and because of this test (that has been added in a batch, for code cov, and not really because it was required), my PR might look like a breaking change.
I am just elaborating, so that we can be aware it might be better not to increase codecov like this, because we adapt use cases (tests) to the code, and it should be the opposite, code should be adapted to a uses cases (tests).

Reply:
i think it's fine anyway. the change is in fact breaking right? just that we don't care if internal behavior changes. easy enough to note on the new PR that the altered tests are of internal behavior.